test/repl.jl

```diff
@@ -122,6 +122,8 @@ temp_pkg_dir(;rm=false) do project_path; cd(project_path) do;
     mktempdir() do tmp_dev_dir
     withenv("JULIA_PKG_DEVDIR" => tmp_dev_dir) do
         pkg"develop Example"
+        pkg"develop Example,PackageCompiler"
+        pkg"develop Example PackageCompiler"
```

Comment:
```suggestion
        pkg"develop Example1, Example2"
        pkg"develop Example1 Example2"
```

Reply:
I thought the use of `Example` relied on this package: https://github.com/JuliaLang/Example.jl i.e. that these had to actually be real packages. Very happy to change this if that's not the case, the use of PackageCompiler here is very clunky.

Reply:
It's not a big deal. `Package1 Package2` also works

Reply:
I missed that this was in tests. Thought it was the docstring.. whoops. 

LGTM then